ktor-features/ktor-i18n/jvm/src/io/ktor/i18n/I18n.kt

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright 2014-2020 JetBrains s.r.o and contributors. Use of this source code is governed by the Apache 2.0 license.
+ */
+
+package io.ktor.i18n
+
+import io.ktor.application.*
+import io.ktor.request.*
+import io.ktor.util.*
+import io.ktor.util.pipeline.*
+import java.nio.charset.*
+
+/**
+ * Represents I18n feature and its configuration.
+ *
+ * install(I18n) {
+ *    defaultLanguage = "pt-BR"
+ *    encoding = StandardCharsets.UTF_8
+ * }
+ */
+class I18n(configuration: Configuration) {
+
+    private val defaultLanguage = configuration.defaultLanguage
+
+    private val encoding = configuration.encoding
+
+    /**
+     * I18n configuration. Currently supports [encoding] and [defaultLanguage]
+     *
+     * [defaultLanguage] must follow IETF BCP 47 language tag string specification
+     */
+    class Configuration {
+        var defaultLanguage: String = ""
```

Comment:
Agreed. Will change that.

Reply:
I didn't change that, because ``ResourceBundle`` can handle that by using a default message bundle with no Locale. For instance, if we don't pass any ``defaultLanguage``,  ``ResourceBundle`` can get ``message.properties`` as default file.

WDYT?